### DISCUSSION TITLE
Release v0.1.9

### DISCUSSION THREAD

**Discussion by alvin-reyes** [application-research/estuary#435] [Announcements]
## What's Changed
* uncomment staging bucket check by @en0ma in https://github.com/application-research/estuary/pull/387
* chore: revert filename to name by @alvin-reyes in https://github.com/application-research/estuary/pull/389
* Constants package by @gmelodie in https://github.com/application-research/estuary/pull/381
* Quality of life changes for set up script by @snissn in https://github.com/application-research/estuary/pull/397
* chore: fix constant variable calls by @alvin-reyes in https://github.com/application-research/estuary/pull/400
* Add build and deploy workflows by @gmelodie in https://github.com/application-research/estuary/pull/401
* fix: application crashing on unverified client datacap by @arihantdaga in https://github.com/application-research/estuary/pull/406
* fix pinning status for shuttle contents by @en0ma in https://github.com/application-research/estuary/pull/378
* Correctly set data transfer failed status and message  by @en0ma in https://github.com/application-research/estuary/pull/377
* chore: added shuttle read-timeout config by @alvin-reyes in https://github.com/application-research/estuary/pull/409
* chore: sync dev with latest from master - 08232022 by @alvin-reyes in https://github.com/application-research/estuary/pull/412
* staging bucket and content manager refactor - part 1 by @en0ma in https://github.com/application-research/estuary/pull/370
* feat: estuary modules/deps version bump by @alvin-reyes in https://github.com/application-research/estuary/pull/413
* Add basic usage docs to README.md by @travis in https://github.com/application-research/estuary/pull/414
* chore: sync dev with latest from master - 08242022  by @alvin-reyes in https://github.com/application-research/estuary/pull/416
* Add --disable-swagger-endpoint CLI flag by @gmelodie in https://github.com/application-research/estuary/pull/422
* clarify difference between --no-storage-cron and --disable-deal-making by @toastts in https://github.com/application-research/estuary/pull/388
* Add /public/miners/storage/query/:miner route to handleQueryAsk by @gmelodie in https://github.com/application-research/estuary/pull/423
* correctly check deal status and correctly restart transfers by @en0ma in https://github.com/application-research/estuary/pull/415
* Do not select shuttle if shuttle content adding is disabled by @en0ma in https://github.com/application-research/estuary/pull/427
* Make collections api restful by @gmelodie in https://github.com/application-research/estuary/pull/426
* Document content/add response structure by @RangerMauve in https://github.com/application-research/estuary/pull/418
* Fix swagger annotation on delete miner endpoint by @gmelodie in https://github.com/application-research/estuary/pull/417

## New Contributors
* @snissn made their first contribution in https://github.com/application-research/estuary/pull/397
* @arihantdaga made their first contribution in https://github.com/application-research/estuary/pull/406
* @travis made their first contribution in https://github.com/application-research/estuary/pull/414
* @toastts made their first contribution in https://github.com/application-research/estuary/pull/388
* @RangerMauve made their first contribution in https://github.com/application-research/estuary/pull/418

**Full Changelog**: https://github.com/application-research/estuary/compare/v0.1.8...v0.1.9

<hr /><em>This discussion was created from the release <a href='https://github.com/application-research/estuary/releases/tag/v0.1.9'>v0.1.9</a>.</em>

**Comment by alvin-reyes** [comment]
# Important notes

- This is the first release that will not be installed/deployed to shuttle-4 and shuttle-5. This means that we need to remove the content adding on these shuttles. As a primary option now, users who needs to upload should use `https://upload.estuary.tech/content/add`.
- The reason why we can't install on shuttle-4 and 5 is due to the version bumps. The version bump requires us to use a specific filecoin-ffi commit [hash](https://github.com/filecoin-project/filecoin-ffi/tree/32afd6e1f1419b6bb7d0f4b3944287fde593ca64) which doesn't compile on arm64 based operating system. Shuttle-4 and 5 are both arm64.
- I confirmed with @lanzafame that shuttles-4 and 5 are just for experimentation purposes. Given that this release has significant importance in the stability of estuary, I'm taking the route of not using arm64 operating system for new shuttles and just make shuttle-4, 5 as download/retrieval gateways with content-adding disabled. Slack discussion [here](https://filecoinproject.slack.com/archives/C016APFREQK/p1663152584062789).
- Moving forward, we will only update the api node, shuttles 1,2,6,7 and 8.

![image](https://user-images.githubusercontent.com/4479171/190143144-8474968c-c951-40ac-974d-5f5156fa70ee.png)

**Reply by gmelodie** [reply]
To add on to that, we do plan to eventually update shuttles 4 and 5, but only when filecoin-ffi starts offering support for arm64.
